packages/composables/src/useCart.test.ts

```diff
@@ -0,0 +1,66 @@
+import { describe, expect, it } from "vitest";
+import { useCart } from "./useCart";
+import { useSetup } from "./_test";
+import type { operations } from "#shopware";
+
+describe("useCart", () => {
+  it("adding products", async () => {
```

Comment:
can we make it more per unit and test `addProduct` and `addProducts` separately?

Reply:
done